packages/ecc-client-ga4gh-trs/src/components/trs-list/definition/trs-list.template.ts

```diff
@@ -0,0 +1,442 @@
+                ></path>
+              </svg>
+              The organization in the registry that published the image.
+            </custom-tooltip>
+          </div>
+          <fast-text-field
+            type="text"
+            name="organization"
+            value="${(x) => x.filterParams.organization}"
+            @input="${(x, c) => x.handleFilterParamChange(c.event)}"
+          ></fast-text-field>
+        </label>
+        <label class="filterContainer--label">
+          <div>
+            <span>Image Name </span>
+            <custom-tooltip>
+              <svg
+                slot="label"
+                class="tooltipIcon"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 16 16"
+                width="16"
+                height="16"
+              >
+                <path
+                  d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8Zm8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13ZM6.5 7.75A.75.75 0 0 1 7.25 7h1a.75.75 0 0 1 .75.75v2.75h.25a.75.75 0 0 1 0 1.5h-2a.75.75 0 0 1 0-1.5h.25v-2h-.25a.75.75 0 0 1-.75-.75ZM8 6a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"
+                ></path>
+              </svg>
+              The name of the image.
+            </custom-tooltip>
+          </div>
+          <fast-text-field
+            type="text"
+            name="name"
+            value="${(x) => x.filterParams.name}"
+            @input="${(x, c) => x.handleFilterParamChange(c.event)}"
+          ></fast-text-field>
+        </label>
+        <label class="filterContainer--label">
+          <div>
+            <span>Author</span>
+            <custom-tooltip>
+              <svg
+                slot="label"
+                class="tooltipIcon"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 16 16"
+                width="16"
+                height="16"
+              >
+                <path
+                  d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8Zm8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13ZM6.5 7.75A.75.75 0 0 1 7.25 7h1a.75.75 0 0 1 .75.75v2.75h.25a.75.75 0 0 1 0 1.5h-2a.75.75 0 0 1 0-1.5h.25v-2h-.25a.75.75 0 0 1-.75-.75ZM8 6a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"
+                ></path>
+              </svg>
+              The author of the tool.
+            </custom-tooltip>
+          </div>
+          <fast-text-field
+            type="text"
+            name="author"
+            value="${(x) => x.filterParams.author}"
+            @input="${(x, c) => x.handleFilterParamChange(c.event)}"
+          ></fast-text-field>
+        </label>
+      </div>
+      <fast-button
+        class="filterContainer--button"
+        @click="${(x) => x.handleApplyFilter()}"
+        >Apply</fast-button
+      >
+      <fast-button
+        class="filterContainer--clear"
+        @click="${(x) => x.handleClearFilter()}"
+        >Clear</fast-button
+      >
+    </div>
+  `
+)}`;
+
+const accordionTemplate = html<TRS>`
```

Comment:
this looks good, but i think you can separate it into 3 components, the top section, the middle section and the bottom section. it's just a bit long for one component.